vipps-login-api-faq.md

```diff
@@ -271,13 +273,44 @@ Vipps Login from phone number are reserved for such special cases and needs to b
 enabled by Vipps for eligible sale units. If a merchant uses Vipps Login from phone number on
 webpages or in apps used by end-users (on their own device), access to the feature can be withdrawn.
 
-Merchants need to apply for access to Vipps Login from phone number separately by sending an email
-to accessuserinfo@vipps.no. In the email you should specify:
+Merchants need to apply for access to Vipps Login from phone number separately. 
+This is done by sending an email to accessuserinfo@vipps.no and supplying the following information:
+ * Merchant serial number (MSN) for the sale unit for which you will use Vipps Login (this can be retrieved from portal.vipps.no)
+ * Information on how you plan to use Vipps Login from phone number
 
-* Merchant name
-* Organization number
-* Name and number of the sale unit from [portal.vipps.no](https://portal.vipps.no)
-* Information on how you plan to use Vipps Login from phone number
+Vipps Login from phone number is a billed service. As such, we also need some details regarding invoicing:
+ * Recipient name
+ * Recipient email
+ * Invoicing address
+
+## How can I get started with delegatedConsents?
+
+If you plan on using the scope delegatedConsents along with Vipps Login from phone number,
+you will need to supply some information on what consents you will retrieve and how.
+We will then tailor this screen to suit your needs. You can see what the flow looks like at
+[Vipps Login from phone number](vipps-login-from-phone-number-api-howitworks.md).
+Send an email to accessuserinfo@vipps.no and supply the following information:
+
+| Item                                                 | Description                                                                                                                                                                    | Example                                                                           | Comments                                                                                               |
+|------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------|
```

Comment:
Hi! It will render it ok even if you don't include all the dashes here. If they become annoying, you can remove some of them.